packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -192,7 +192,7 @@ Host platform code is in the android/ and ios/ directories under $relativePlugin
       String projectDescription, String dirPath, String flutterRoot,
       { bool renderDriverTest: false, bool withPluginHook: false }) {
     flutterRoot = fs.path.normalize(flutterRoot);
-    flutterRoot = _relativePath(from: dirPath, to: flutterRoot);
+    final String relativeFlutterRoot = _relativePath(from: dirPath, to: flutterRoot);
```

Comment:
Is `flutterRoot` guaranteed to be an absolute path? We normalise it but don't pass it through `absolute` here.